### PR TITLE
Fix issue #198

### PR DESCRIPTION
**Pull request by saledouble**
Related to #198.

### CODE REVIEW

**Comment by saledouble** [comment]
New EV is `edu.cuny.hunter.log.evaluation.notLowerLogLevelInIfStatement`.
https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/wiki/Running-the-Evaluator

File: edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java [hunk 1]
```diff
@@ -369,13 +384,28 @@ private Boolean checkIfBlock(ASTNode node) {
 						|| condition.contains("DEBUG") || condition.contains("TRACE")) {
 					LOGGER.info("We meet a logging wrapping: \n" + node);
 					return true;
-				}
+				} else
+                                    return false;
```
Comment (khatchad) [inline edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java]: @saledouble I don't think we want to change this:
1. We are trying to avoid big changes at this point.
1. We want to keep going up to see other `if` statements to see if there is wrapping.

Comment (saledouble) [reply edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java]: I see.

File: edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java [hunk 2]
```diff
@@ -369,13 +384,27 @@ private Boolean checkIfBlock(ASTNode node) {
 						|| condition.contains("DEBUG") || condition.contains("TRACE")) {
 					LOGGER.info("We meet a logging wrapping: \n" + node);
 					return true;
-				}
+				}
```
Comment (khatchad) [inline edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java]: Why this change?

File: edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java [hunk 3]
```diff
@@ -369,13 +384,27 @@ private Boolean checkIfBlock(ASTNode node) {
 						|| condition.contains("DEBUG") || condition.contains("TRACE")) {
```
Comment (khatchad) [inline edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java]: Shouldn't you ignore case here?

Comment (saledouble) [reply edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java]: I ignored. When it returns true, the transformation action will be NONE and the tool will count them.

File: edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java (hunk 2 above)
Comment (saledouble) [reply edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java]: Fixed.
